### PR TITLE
Adds New pathway

### PR DESCRIPTION
**Pull request by wivern**

### CODE REVIEW

File: src/main/resources/r/run_pathways.R [hunk 1]
```diff
@@ -0,0 +1,163 @@
+library(DatabaseConnector)
+library(SqlRender)
+library(rlist)
+library(purrr)
+library(dplyr)
+
+getEventCohortCodes <- function(pathwayAnalysis) {
```
Comment (pavgra) [inline src/main/resources/r/run_pathways.R]: THe R script should be as simple as the Cohort Characterization one. Seems like we need to extract Pathways logic to a separate R package

File: src/main/java/com/odysseusinc/arachne/portal/util/SubmissionHelper.java [hunk 2]
```diff
@@ -421,6 +432,94 @@ public void updateExtendInfo(Submission submission) {
         }
     }
 
+    private class PathwaySubmissionExtendInfoStrategy extends SubmissionExtendInfoAnalyzeStrategy {
+
+        @Override
+        public void updateExtendInfo(Submission submission) {
```
Comment (YaroslavTir) [inline src/main/java/com/odysseusinc/arachne/portal/util/SubmissionHelper.java]: this method is too long, can you split it into a few? like `getPathwayCodes` etc

File: src/main/java/com/odysseusinc/arachne/portal/util/SubmissionHelper.java [hunk 3]
```diff
@@ -421,6 +432,94 @@ public void updateExtendInfo(Submission submission) {
...
+                }, ArachneCollectors.toJsonArray());
+
+                // Cohort Paths
+                path = filePath(resultsDir, "pathway_results.csv");
+                Map<Integer, JsonArray> pathwayResults  = parseCsv(path, rec -> {
+                   JsonObject result = new JsonObject();
+                   result.add("targetCohortId", getJsonPrimitive(rec.get("TARGET_COHORT_ID")));
+                   result.add("personCount", getJsonPrimitive(rec.get("COUNT_VALUE")));
+                   List<String> cohortPath = new ArrayList<>();
+                   for(int i = 1; i <= 10; i++) {
```
Comment (YaroslavTir) [inline src/main/java/com/odysseusinc/arachne/portal/util/SubmissionHelper.java]: magic number, if it is really necessary to keep it, then create a constant with descriptive name

File: src/main/resources/r/run_pathways.R [hunk 4]
```diff
@@ -0,0 +1,163 @@
+library(DatabaseConnector)
+library(SqlRender)
+library(rlist)
+library(purrr)
```
Comment (YaroslavTir) [inline src/main/resources/r/run_pathways.R]: this file is part of the, it is not necessary to keep it here.

**Pull request merged**